Volume/Views/Onboarding/WelcomeView.swift

```diff
@@ -13,12 +13,12 @@ extension OnboardingView {
         var body: some View {
             Group {
                 FeatureRow(
-                    image: "publications",
+                    image: "pen",
```

Comment:
TODO for self: change `Image` API to provide raw strings for situations like this